Content.Client/Radar/RadarConsoleWindow.xaml.cs

```diff
@@ -66,20 +66,20 @@ protected override void Draw(DrawingHandleScreen handle)
         var fakeAA = new Color(0.08f, 0.08f, 0.08f);
         var gridLines = new Color(0.08f, 0.08f, 0.08f);
         var gridLinesRadial = 8;
-        var gridLinesEquatorial = 8;
+        var gridLinesEquatorial = (int) Math.Floor(_radarRange / GridLinesDistance);
 
-        handle.DrawCircle((point, point), RadiusCircle + 1, fakeAA);
-        handle.DrawCircle((point, point), RadiusCircle, Color.Black);
+        handle.DrawCircle((point, point), ScaledMinimapRadius + 1, fakeAA);
+        handle.DrawCircle((point, point), ScaledMinimapRadius, Color.Black);
 
-        for (var i = 0; i < gridLinesEquatorial; i++)
+        for (var i = 1; i < gridLinesEquatorial + 1; i++)
         {
-            handle.DrawCircle((point, point), (RadiusCircle / gridLinesEquatorial) * i, gridLines, false);
+            handle.DrawCircle((point, point), GridLinesDistance * MinimapScale * i, gridLines, false);
         }
 
         for (var i = 0; i < gridLinesRadial; i++)
         {
             Angle angle = (Math.PI / gridLinesRadial) * i;
-            var aExtent = angle.ToVec() * RadiusCircle;
+            var aExtent = angle.ToVec() * ScaledMinimapRadius;
             handle.DrawLine((point, point) - aExtent, (point, point) + aExtent, gridLines);
         }
```

Comment:
Looks like these don't draw correctly anymore.

Reply:
I see them draw correctly, the screenshots show them too.